src/problem_solvers.jl

```diff
@@ -9,6 +9,7 @@ using ..Options
 
 using NamedTrajectories
 using MathOptInterface
+using Ipopt
 const MOI = MathOptInterface
 
```

Comment:
would be nice to add docstring for solve :)

Reply:
Definitely, just noting it here - adding some features to save best trajectory (on various parameters), access to variables saved inside of callbacks programmatically (`get_callbacks`)